### ISSUE TITLE
Microsoft Edge excluded with latest modern_rules

### ISSUE THREAD

**Issue by bmclean** [fnando/browser#430]
## Description

It seems that the latest `modern_rules` excludes the current version of Microsoft Edge.

## How to reproduce

The rule is defined here:
```
rules << ->(b) { b.edge? && b.version.to_i >= 39 && !b.compatibility_view? }
```
The current release of Microsoft Edge has a user agent that looks like this:

`Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36 Edge/18.18363`

and the `b.version.to_i` is equal to 18

## Software:
- Browser gem version: 3.0.0

**Comment by fnando** [comment]
Good catch! I'm pushing a new release to fix this. Does this look good?

```diff
diff --git a/lib/browser/browser.rb b/lib/browser/browser.rb
index 2e44502..8fd8953 100644
--- a/lib/browser/browser.rb
+++ b/lib/browser/browser.rb
@@ -96,7 +96,7 @@ module Browser
     rules << ->(b) { b.safari? && b.version.to_i >= 10 }
     rules << ->(b) { b.firefox? && b.version.to_i >= 52 }
     rules << ->(b) { b.ie? && b.version.to_i >= 11 && !b.compatibility_view? }
-    rules << ->(b) { b.edge? && b.version.to_i >= 39 && !b.compatibility_view? }
+    rules << ->(b) { b.edge? && b.version.to_i >= 18 && !b.compatibility_view? }
     rules << ->(b) { b.opera? && b.version.to_i >= 50 }
   end
```

**Comment by fnando** [comment]
That's one of the reasons I'd rather not a have definition of what's modern baked into this library. I'm thinking about releasing yet another major version with `rules << ->(_b) { true }`, which would return every client as a modern browser.

If somebody cares enough about this, I'd recommend creating a separate gem to define theses rules, e.g. `browser-modern-rules`.

**Comment by bmclean** [comment]
You likely don't need the `!b.compatibility_view?` for Edge. The only option I see in Edge is to open the page using Internet Explorer which would then fallback to the `ie?` rules. I guess the question is what defines a modern version of Microsoft Edge? Perhaps `>= 15` is the right answer. That would represent version 40 which was released April 11, 2017

**Comment by fnando** [comment]
@bmclean #431 is up. Can you review it?

**Comment by fnando** [comment]
Thanks, @bmclean. I just released 3.0.1.

**Comment by bmclean** [comment]
You're welcome. Regarding removing the baked in definition rules of modern, I would be fine with that. We have different rules defined for some of them already anyway.
